lib/autoupdate/start.rb

```diff
@@ -110,8 +110,20 @@ def start(interval:, args:)
       set_env << "\nexport SUDO_ASKPASS=#{env_sudo}"
     end
 
+    ac_only = if args.ac_only?
+      <<~EOS
+        if [[ $(pmset -g ps | head -1) =~ "Battery Power" ]]; then
+          echo "Not starting autoupdate, because device is running on Battery"
```

Comment:
Does this print to the applet notification? Will the user be notified essentially that Autoupdate hasn't actually been run?

Reply:
The whole `autoupdate` script will not run, so no AppletSvript will be invoked.

Reply:
Does the `autoupdate` script _ever_ start or is it essentially "stuck" until the user tries to activate it again? That's my only concern here.

Reply:
This time, I'm sorry for the late reply, totally missed it.

It will only run, if the user ever again plugs in power (what he's very likely to do since he want's to continue using the device) and another trigger (reboot or time) initiates the execution of the autoupdate script.